jeremy2918/tsseract
# Feat/create post

# [Create Post [Client]](https://trello.com/c/iA8mgsaq/27-i-want-to-build-the-create-post-functionality-in-the-app-to-be-able-to-write-preview-and-publish-a-post-from-the-website)

## Description
This PR adds several components that combined end up creating the user interface to create a post. It allows the user to write a post title and body, choose a cover image from the web and add emojis and more images to the body of the post. The user can also preview the post he or she is creating.
This branch also adds a simple functionality to switch between light and dark mode.


## Testing Steps

- `git pull`
- `git checkout feat/create-post`
- `npm install`
- `npm run tsc`
- `npm run dev:server`
- Go to `https://localhost:3000` and you should see the page there.
- Write a title and you should be able to preview it switching the view. 
- Do the same with the content, you can write in markdown and add emojis.
- Try adding a cover image and images to the content of the page.
